tests/pipelines/test_pipelines_token_classification.py

```diff
@@ -561,7 +561,18 @@ def test_aggregation_strategy(self):
             },
             {
                 # fmt: off
-                "scores": np.array([0, 0, 0, 0, 0, 0.9986497163772583, 0, 0, ]),
+                "scores": np.array(
+                    [
+                        0,
+                        0,
+                        0,
+                        0,
+                        0,
+                        0.9986497163772583,
+                        0,
+                        0,
+                    ]
+                ),
```

Comment:
Guess this is not what we love to have, and related to Lysandre's comment I believe